packages/web/pages/technology/[id]/edit.js

```diff
@@ -108,6 +108,17 @@ const TechnologyFormPage = ({ taxonomies, technology }) => {
 	} = router;
 	const [submitting, setSubmitting] = useState(false);
 
+	/**
+	 * We use 'undefined' as a status because this file is reused in new technology page
+	 * So we can't deny user access if it's a new register
+	 */
+	const authorizedToEdit = [
+		undefined,
+		statusEnum.DRAFT,
+		statusEnum.REQUESTED_CHANGES,
+		statusEnum.PUBLISHED,
+	].includes(technology?.status);
```

Comment:
```suggestion
	const authorizedToEdit = !technology || [
		statusEnum.DRAFT,
		statusEnum.REQUESTED_CHANGES,
		statusEnum.PUBLISHED,
	].includes(technology?.status);
```

Reply:
Can't we achieve the same with this code? without the undefined? or something similar, I think this is much preferable than have the `undefined` here

Reply:
I totally agree with you, will implement this